### ISSUE TITLE
Deoplete show completions only after source file is saved

### ISSUE THREAD

**Issue by Silnar** [racer-rust/vim-racer#37]
Hi,

I've tried to use vim-racer with deoplete but it only show completions after the buffer with source code is saved.

For example, when I open below code:

```
fn main() {
    let v: Vec<_> = vec![1, 2, 3];
}
```

and start writing new line: `v.as` - it doesn't show any completions.

But when I write:

```
fn main() {
    let v: Vec<_> = vec![1, 2, 3];
    v.
}
```

and save it, then it will show completions when I start writing after the dot.

Can you help?
Cheers.

**Comment by Shougo** [comment]
Fixed by #38.

**Comment by Silnar** [comment]
Super!
Thanks :)
